<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qgisDemoClass</class>
 <widget class="QMainWindow" name="qgisDemoClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qgisDemo</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>开始</string>
    </property>
    <addaction name="actionaddVectorLayer"/>
    <addaction name="actionaddRasterLaer"/>
    <addaction name="actionlayerProperty"/>
    <addaction name="actionstyleManager"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionaddVectorLayer">
   <property name="text">
    <string>添加矢量图层</string>
   </property>
  </action>
  <action name="actionaddRasterLaer">
   <property name="text">
    <string>添加栅格图层</string>
   </property>
  </action>
  <action name="actionlayerProperty">
   <property name="text">
    <string>属性表</string>
   </property>
  </action>
  <action name="actionstyleManager">
   <property name="text">
    <string>风格管理器</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qgisDemo.qrc"/>
 </resources>
 <connections/>
</ui>
